pkg/pod/entrypoint.go

```diff
@@ -57,6 +57,7 @@ const (
 	readyAnnotation        = "tekton.dev/ready"
 	readyAnnotationValue   = "READY"
 
+	// stepPrefix is the prefix applied to the container name of a step container.
```

Comment:
this comment seems unrelated to this change

also if we add it, then we should add one for sidecarPrefix too I think

Reply:
yes. Fixing.